src/riak_kv_vnode.erl

```diff
@@ -2598,47 +2598,40 @@ non_neg_env(App, EnvVar, Default) when is_integer(Default),
 %% 2. local found, but never acted on this key before (or don't remember it!)
 %% 3. local found, local acted, but incoming has greater count or actor epoch
 %%    This one is tricky, since it indicates some byzantine failure somewhere.
--spec maybe_new_key_epoch(boolean(), #state{},
-                          riak_object:riak_object(),
-                          riak_object:riak_object()) ->
-                                 {binary(), #state{}}.
-maybe_new_key_epoch(false, State, _, _) ->
-    %% Never add a new key epoch when not coordinating
-    %% @TODO (rdb) need to mark actor as dirty though.
-    {State#state.vnodeid, State};
-maybe_new_key_epoch(true, State=#state{counter=#counter_state{use=false}, vnodeid=VId}, _, _) ->
+-spec maybe_new_key_epoch(Coord::boolean(), State::#state{},
+                          LocalObject::riak_object:riak_object(),
+                          IncomingObject::riak_object:riak_object()) ->
+                                 {ActorId:: binary(), #state{}}.
```

Comment:
Thanks for adding the variable names to the type specs - definitely improves readability in many places.

In the case of `ActorId` I see we specify `ActorId :: binary()` in a couple places. Maybe worth defining a local `actor_id()` type in this module to avoid redundancy?